network/ServerNetworking.cpp

```diff
@@ -212,8 +212,11 @@ void PlayerConnection::SetCookie(boost::uuids::uuid cookie)
 const std::string& PlayerConnection::ClientVersionString() const
 { return m_client_version_string; }
 
-bool PlayerConnection::ClientVersionStringMatchesThisServer() const
-{ return !m_client_version_string.empty() && m_client_version_string == FreeOrionVersionString(); }
+bool PlayerConnection::ClientVersionStringMatchesThisServer() const {
```

Comment:
Need to rename this or move the new logic to where it is used.

Reply:
Is `IsBinarySerializationUsed` correct name?

Reply:
Changed to new name.